packages/terra-alert/docs/MoreInformation.md

```diff
@@ -20,7 +20,7 @@ This image shows the layout of the elements of the Alert component.
 |action |element  |optional  |null  |An action element to be added to the action section of the alert.|
```

Comment:
I think that we could actually just remove this MoreInformation file. All this info is generated on the dev site.

Reply:
fixed here https://github.com/cerner/terra-core/pull/2407/commits/ffe3ff377f99567cc3e6d1d5c1be4eb714f01fe1.